### ISSUE TITLE
['mod', 'rst', 'sharp', 'template'] fail to load. 

### ISSUE THREAD

**Issue by Carreau** [grantjenks/py-tree-sitter-languages#15]
They all give a similar error:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In [24], line 1
----> 1 get_language(fail[3])

File tree_sitter_languages/core.pyx:14, in tree_sitter_languages.core.get_language()

File ~/miniforge3/envs/papyri/lib/python3.9/site-packages/tree_sitter/__init__.py:83, in Language.__init__(self, library_path, name)
     81 self.name = name
     82 self.lib = cdll.LoadLibrary(library_path)
---> 83 language_function = getattr(self.lib, "tree_sitter_%s" % name)
     84 language_function.restype = c_void_p
     85 self.language_id = language_function()

File ~/miniforge3/envs/papyri/lib/python3.9/ctypes/__init__.py:395, in CDLL.__getattr__(self, name)
    393 if name.startswith('__') and name.endswith('__'):
    394     raise AttributeError(name)
--> 395 func = self.__getitem__(name)
    396 setattr(self, name, func)
    397 return func

File ~/miniforge3/envs/papyri/lib/python3.9/ctypes/__init__.py:400, in CDLL.__getitem__(self, name_or_ordinal)
    399 def __getitem__(self, name_or_ordinal):
--> 400     func = self._FuncPtr((name_or_ordinal, self))
    401     if not isinstance(name_or_ordinal, int):
    402         func.__name__ = name_or_ordinal

AttributeError: dlsym(0x2089543a0, tree_sitter_<TARGET LANGUAGE HERE>): symbol not found
```

This is using 1.5.0 – tree_sitter_languages-1.5.0-cp39-cp39-macosx_11_0_arm64.whl

This should be exhaustive of the languages that don't load.

**Comment by grantjenks** [comment]
I'm not sure what "mod" refers to? If it means "go-mod" then that can be loaded as:
```
In [9]: language = get_language('gomod')

In [10]: parser = get_parser('gomod')
```

**Comment by grantjenks** [comment]
Similarly, "sharp" maybe refers to C#? That can be loaded as:
```
In [13]: parser = get_parser('c_sharp')

In [14]: language = get_language('c_sharp')
```

**Comment by grantjenks** [comment]
I think "template" refers to "embedded_template" like:
```
In [15]: parser = get_parser('embedded_template')

In [16]: language = get_language('embedded_template')
```

**Comment by grantjenks** [comment]
Missing rst was a mistake. Fixed in 0fbabda. To be released in 1.6.0
